backend/app/app.go

```diff
@@ -110,6 +111,7 @@ func Start(
 		dataDogAPIKey,
 		segmentAPIKey,
 		ipStackAPIKey,
+		config.SearchAPITimeout,
```

Comment:
```suggestion
		provider.SearchAPITimeout(config.SearchAPITimeout),
```
Wire injects dependencies based on the type of the parameter.